### ISSUE TITLE
Reproduce Things and Sintel

### ISSUE THREAD

**Issue by mmendiet** [DQiaole/MemFlow#15]
Hello! Thank you for your excellent work! I have been attempting to reproduce the MemFlowNet_things.pth and MemFlowNet_sintel.pth using the training script. I'm using a very similar setup, same number of GPUs and batch size, etc. However, when I use the training commands in train.sh, I am having trouble reproducing the paper results. Just wanted to check here and see if I am missing something :)

For things, I start with the provided skflow-things.pth. But currently, during the first 30k iterations, then training is showing the model to get progressively worse. See the training EPE and 1px accuracy below.

<img width="927" alt="Image" src="https://github.com/user-attachments/assets/1c1968ed-f4bd-46dc-a335-0a6c276adf87" />

<img width="928" alt="Image" src="https://github.com/user-attachments/assets/e863d7c3-96e6-4601-8945-77942a5cbc5a" />

Do you see this trend as well early in the training? Are there any details to keep in mind to improve the optimization?

For sintel, I start with the provided MemFlowNet_things.pth. This one does seem to head towards convergence. However, it seems I'm not getting as good of a fit as the provided pretrained model. For example, the provided MemFlowNet_sintel.pth gets  {'final': 0.495351, 'clean': 0.35456997} on sintel_train, but my best fully trained 600k model so far gets {'final': 0.59550947, 'clean': 0.41282004}. I can't seem to bridge the gap between the provided checkpoints and the attempted reproduction. Please let me know if you have any suggestions, thank you!

**Comment by DQiaole** [comment]
Hi, during the initial training stage of things, it is normal to observe the degradation of performance, as we introducing additional temporal information:

<img width="345" alt="Image" src="https://github.com/user-attachments/assets/a0708e1f-1486-4466-8edf-da207a49591a" />

As for sintel, we indeed achieved a epe of 0.49 within 600k steps with the same config:

<img width="371" alt="Image" src="https://github.com/user-attachments/assets/f99968b0-1fd5-48fd-a43a-243085efc161" />

so, I recommend you to add more training steps if you want to achieve a small training error.
